examples/bigquery_user_info_updater_tool/bigquery_user_info_updater/update_user_info.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import argparse
+import logging
+
+from updater_tools import query_creator
+from updater_tools import user_info_updater
+
+
+def parse_args(argv):
+    """Parses arguments from command line.
+
+    Args:
+        argv: list of arguments.
+
+    Returns:
+        parsed_args: parsed arguments.
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--schema_path',
+        help='Path to the user tables schema.',
+        default=True
+    )
+    parser.add_argument(
+        '--project_id',
+        help='Id of project holding BQ resources.',
+        required=True
+    )
+    parser.add_argument(
+        '--dataset_id',
+        help='Id of dataset holding Identity BQ tables.',
+        required=True
+    )
+    parser.add_argument(
+        '--final_table_id',
+        help='Id of final table holding golden record for each user.',
+        required=True
+    )
+    parser.add_argument(
+        '--updates_table_id',
+        help='Id of table holding user updates.',
+        required=True
+    )
+    parser.add_argument(
+        '--temp_updates_table_id',
+        help='Id of table that deduped update rows will be written to.',
+        required=True
+    )
+    parser.add_argument(
+        '--user_id_field_name',
+        help='Name of the field that identifies unique users.',
+        required=True
+    )
+    parser.add_argument(
+        '--ingest_timestamp_field_name',
+        help='Name of the timestamp field that marks the ingestion.'
+             'of user rows.',
+        required=True
+    )
+
+    args = parser.parse_args(args=argv)
+    return args
+
+
+def main(argv=None):
+    """Main class for updating user info.
+
+    args:
+        argv: Args from command line.
+    """
+    args = parse_args(argv)
+    schema_path = args.schema_path
+    project_id = args.project_id
+    dataset_id = args.dataset_id
+    final_table_id = args.final_table_id
+    updates_table_id = args.updates_table_id
+    temp_updates_table_id = args.temp_updates_table_id
+    user_id_field_name = args.user_id_field_name
+    ingest_timestamp_field_name = args.ingest_timestamp_field_name
+
+    # Create queries needed to run the updates
```

Comment:
jaketf on Aug 27, 2019 Member
this comment implies that UserInfoUpdater.__init__ is starting a process but I don't think it does.
remove?